src/nvim/ex_getln.c

```diff
@@ -3875,7 +3875,9 @@ expand_shellcmd (
     STRLCPY(buf + l, pat, MAXPATHL - l);
 
     /* Expand matches in one directory of $PATH. */
+#ifndef NDEBUG
     char_u **prev_file = *file;
+#endif
```

Comment:
This was introduced because of a coverity warning. Would be removed with https://github.com/oni-link/neovim/commit/635042b6829b7fe36f0aee449929751aa5143f70